app/views/pages/_settings_form.slim

```diff
@@ -6,10 +6,6 @@
       = f.select :tag_ids, options_from_collection_for_select(Tag.all, 'id', 'name', page.tag_ids), {}, html_options= {class: 'selectize-container', multiple: true}
 
     .form-group
-      = f.label :language_id, t('pages.edit.language_label')
-      = f.select :language_id, Language.all.map{|lang| [lang.name, lang.id] }, {}, class: "form-control"
-
-    .form-group
```

Comment:
I don't think this should be removed from the edit page. is this an issue with AK integration?